integration/release_upgrade_linux_test.go

```diff
@@ -49,27 +49,38 @@ type beforeUpgradeHookFunc func(*testing.T)
 
 // TODO: Support Windows
 func TestUpgrade(t *testing.T) {
-	previousReleaseBinDir := t.TempDir()
-	downloadPreviousLatestReleaseBinary(t, previousReleaseBinDir)
-
-	t.Run("recover", runUpgradeTestCase(previousReleaseBinDir, shouldRecoverAllThePodsAfterUpgrade))
-	t.Run("exec", runUpgradeTestCase(previousReleaseBinDir, execToExistingContainer))
-	t.Run("manipulate", runUpgradeTestCase(previousReleaseBinDir, shouldManipulateContainersInPodAfterUpgrade))
-	t.Run("recover-images", runUpgradeTestCase(previousReleaseBinDir, shouldRecoverExistingImages))
-	t.Run("metrics", runUpgradeTestCase(previousReleaseBinDir, shouldParseMetricDataCorrectly))
+	for _, version := range []string{"1.7", "2.0"} {
```

Comment:
Is this the line we'll now need to update for future releases?

Reply:
We should for now, we can define some rules for this and calculate from current version though.